uptane-standard.md

```diff
@@ -429,11 +429,11 @@ The following sections describe the role-specific metadata. All roles SHALL foll
 A repository's Root metadata distributes the public keys of the top-level Root, Targets, Snapshot, and Timestamp roles, as well as revocations of those keys. It SHALL contain two attributes:
 
 * A representation of the public keys for all four roles. Each key SHALL have a unique public key identifier.
-* An attribute mapping each role to (1) its public key(s), and (2) the threshold of signatures required for that role.
+* An attribute mapping each role to (1) its public key(s), and (2) the threshold of signatures required for that role
```

Comment:
Why remove the period? The line before it still has a period. I'd prefer to keep it consistent.

Reply:
Partway through, I think I started questioning if the period was needed. At  this point, as long as we're consistent, I'll put the period back in.

Reply:
Done.